etl/steps/data/garden/shift/2023-07-10/energy_production_from_fossil_fuels.excluded_countries.json

```diff
@@ -0,0 +1,3 @@
+[
+    "Inde"
```

Comment:
No idea! My guess is that by mistake they used the French word for India (given that the creators are French). I think I checked at some point and mapping "Inde" to "India" was problematic because there were duplicates.